static/js/app.js

```diff
@@ -532,3 +532,12 @@ function vpnBannerLogic() {
 
   vpnPromoFunctions.init();
 }
+
+//Micro Survey Banner
+function dismissSurvey() {
+	var survey_banner = document.getElementById("micro-survey-banner");
```

Comment:
We should use `const` (and `let`) instead of `var`. 

```suggestion
	const survey_banner = document.getElementById("micro-survey-banner");
```

Reply:
Done! Ques: why do we use `const` over `var`?

Reply:
It's a convention from ES6. The `const`value  can't be changed through reassignment, and it can't be redeclared. Since it cannot be reassigned, you have less scenarios, side effects or edge cases to consider.